client-v2/src/components/FrontsCAPIInterface/Feed.tsx

```diff
@@ -1,35 +1,81 @@
 import React from 'react';
 import { styled } from 'constants/theme';
 import FeedItem from './FeedItem';
-import { CapiArticle } from 'types/Capi';
-
-interface FeedProps {
-  articles?: CapiArticle[];
-  error: string | null;
-}
+import {
+  liveSelectors,
+  previewSelectors,
+  prefillSelectors
+} from 'bundles/capiFeedBundle';
+import { selectIsPrefillMode } from 'selectors/feedStateSelectors';
+import { State } from 'types/State';
+import { connect } from 'react-redux';
 
 interface ErrorDisplayProps {
-  error: string | null;
+  error?: string;
   children: React.ReactNode;
 }
 
+interface FeedContainerProps {
```

Comment:
This type is very broad. Will we ever have both `liveArticleIds` and `previewArticleIds` and `prefillArticleIds`?

Would something like this be stricter:

```typescript
type Mode = 'prefill' | 'preview' | 'live';

interface FeedContainerProps {
  mode: Mode;
  articleIds: string[];
  error: string | undefined;
}
```

Reply:
Yes, this is a hangover from code previously in `FeedContainer`. I moved it without touching but it's very obviously due a refactor! I'll take a look, I suspect at the time I was keen to avoid expanding the scope of the PR.

Reply:
Ah hadn't acknowledged it was a just a move. Let's keep as is then to keep a small diff.